prod-sample/sample-modules/frank-lloyd-root/0.0.3/src/pwa.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/* eslint-disable global-require */
+
+
+export const defaultIcon = 'http://localhost:3001/static/modules/frank-lloyd-root/0.0.3/assets/pwa-icon-192px.png';
+export const appleIcon = 'http://localhost:3001/static/modules/frank-lloyd-root/0.0.3/assets/pwa-icon-180px.png';
+
+export const scope = '/';
+// eslint-disable-next-line camelcase
+export const start_url = '/success';
```

Comment:
is this used elsewhere? (child routes?)
an editor might help in finding usage, but PR reviewers don't have that editor during the review/comment phase :S I'm guessing there are a few blog posts on the impacts of export location & design on ability to review and maintain 🤔(but I haven't read them recently so I'm trying to be cautious in my critiques 😓)

Reply:
Interesting, would love to hear more about that... I've removed it for now.